tests/agent/nodejs/app.js

```diff
@@ -1,28 +1,42 @@
 'use strict'
 
 var apm = require('elastic-apm').start({
-  // Set required app name (allowed characters: a-z, A-Z, 0-9, -, _, and space)
-  appName: 'nodejs-testapp',
+  appName: 'test-app', flushInterval: 1
+})
 
-  // Use if APM Server requires a token
-  secretToken: '',
 
-  // Set custom APM Server URL (default: http://localhost:8200)
-  serverUrl: '',
+var app = require("express")();
```

Comment:
How about moving it to `tests/agent/nodejs/express/app.js` as we also have it namespaced for `flask` in `python`?

Reply:
@jalvz what do you think about moving this file to a subfolder as for python? 
Apart from that LGTM.

Reply:
yes, sounds good!